### ISSUE TITLE
-Wunused-variable in drivers/video/fbdev/omap2/omapfb/dss/core.c

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/linux#362]
KernelCI is reporting:
```
    drivers/video/fbdev/omap2/omapfb/dss/core.c:172:6: warning: unused variable 'r' [-Wunused-variable]
```

**Comment by nathanchance** [comment]
Anders Roxell sent a patch for this warning: https://lore.kernel.org/lkml/20190213084721.29624-1-anders.roxell@linaro.org/

**Comment by nathanchance** [comment]
Patch accepted: https://github.com/bzolnier/linux/commit/4c83c2f75a8f4737a8fb177bb7f2ffe464f567ca

**Comment by nathanchance** [comment]
Fixed in `next-20190304`: https://git.kernel.org/next/linux-next/c/4c83c2f75a8f4737a8fb177bb7f2ffe464f567ca
